integration_tests/e2e/support-needs.cy.ts

```diff
@@ -0,0 +1,199 @@
+context('Support Needs', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+    cy.task('stubJohnSmithGetAccommodation')
+  })
+
+  it('Accommodation pathway page should show no support needs set', () => {
+    cy.task('stubJohnSmithGetAccommodationNoSupportNeeds')
+    cy.signIn()
+
+    cy.visit('/accommodation/?prisonerNumber=A8731DY')
+
+    cy.get('.app-summary-card__title').should('contain.text', 'Support needs')
+
+    cy.get('.govuk-summary-card__actions').should('have.attr', 'href')
+    cy.get('.govuk-summary-card__actions').should('contain.text', 'Add a support need')
+
+    cy.get('.app-summary-card__body > p').should('contain.text', 'Support needs not set')
+  })
+
+  it('Adding an Accommodation support need', () => {
+    cy.task('stubJohnSmithGetAccommodationNoSupportNeeds')
+    cy.signIn()
+
+    cy.visit('/accommodation/?prisonerNumber=A8731DY')
+
+    // Click 'Add Support Need'
+    cy.get('.govuk-summary-card__actions').click()
+
+    cy.get('.govuk-grid-column-full > h1').should('contain.text', 'Add accommodation support needs')
+
+    // Click to expand 'support needs already selected - verify 'No support needs selected'
+    cy.get('.govuk-details__summary').click()
+    cy.get('.govuk-details__text > p').should('contain.text', 'No support needs selected')
+
+    // Fill in support needs form
+    // Tick 'End a tenancy'
+    cy.get('.govuk-checkboxes__input[name="support-need-option-Accommodation before custody"]').first().check()
+    // Tick 'No new accommodation support needs identified'
+    cy.get('.govuk-checkboxes__input[name="support-need-option-Moving to new accommodation"]').eq(3).check()
+    // Tick 'Set up payment for rent arrears'
+    cy.get('.govuk-checkboxes__input[name="support-need-option-Accommodation related debt and arrears"]')
+      .first()
+      .check()
+    // Tick 'Other' and fill in custom support need.
+    cy.get('.govuk-checkboxes__input[name="support-need-option-Accommodation related debt and arrears"]').eq(2).check()
+    cy.get('.govuk-input').eq(2).type('Custom accommodation support need')
+
+    // Click Continue
+    cy.get('.govuk-button[type="submit"]').click()
+    cy.get('.govuk-grid-column-three-quarters > h1').should('contain.text', 'End a tenancy')
+
+    // Test back button
+    cy.get('.govuk-back-link').click()
+    cy.get('.govuk-grid-column-full > h1').should('contain.text', 'Add accommodation support needs')
+    cy.get('.govuk-button[type="submit"]').click()
+
+    // Fill in 'End a tenancy' support need
+    cy.get('[id="status"]').check()
```

Comment:
Can you use standard css selectors in cypress? Thinking `.get('#status')` may be simpler and possibly more performant?